metricflow/test/model/validations/test_configurable_rules.py

```diff
@@ -1,43 +0,0 @@
-import pytest
-
-from metricflow.dataset.dataset import DataSet
-from metricflow.model.model_validator import ModelValidator
-from metricflow.model.validations.materializations import ValidMaterializationRule
-from dbt_semantic_interfaces.objects.user_configured_model import UserConfiguredModel
-from metricflow.test.model.validations.helpers import materialization_with_guaranteed_meta
-from metricflow.test.test_utils import model_with_materialization
-
-
-def test_can_configure_model_validator_rules(  # noqa: D
-    simple_model__with_primary_transforms: UserConfiguredModel,
-) -> None:
-    model = model_with_materialization(
-        simple_model__with_primary_transforms,
-        [
-            materialization_with_guaranteed_meta(
-                name="foobar",
-                metrics=["invalid_bookings"],
-                dimensions=[DataSet.metric_time_dimension_name()],
-            )
-        ],
-    )
-
-    # confirm that with the default configuration, an issue is raised
-    issues = ModelValidator().validate_model(model).issues
-    assert len(issues.all_issues) == 1, f"ModelValidator with default rules had unexpected number of issues {issues}"
-
-    # confirm that a custom configuration excluding ValidMaterializationRule, no issue is raised
-    rules = [rule for rule in ModelValidator.DEFAULT_RULES if rule.__class__ is not ValidMaterializationRule]
```

Comment:
I think this can be changed to the following
```
    model = copy.deepcopy(simple_model__with_primary_transforms)
    model.metrics.append(metric_with_guaranteed_meta(
        name='metric_doesnt_exist_squared',
        type=MetricType.DERIVED,
        type_params=MetricTypeParams(
            expr='metric_doesnt_exist * metric_doesnt_exist',
            metrics=[MetricInput(name='metric_doesnt_exist')]
        ),
    ))

    # confirm that with the default configuration, an issue is raised
    issues = ModelValidator().validate_model(model).issues
    assert len(issues.all_issues) == 1, f"ModelValidator with default rules had unexpected number of issues {issues}"

    # confirm that a custom configuration excluding ValidMaterializationRule, no issue is raised
    rules = [rule for rule in ModelValidator.DEFAULT_RULES if rule.__class__ is not DerivedMetricRule]
```

Reply:
Added back!

Reply:
Thank you! ❤️